lisa/tests/scheduler/load_tracking.py

```diff
@@ -694,51 +636,126 @@ def item_test(test_item):
             )
         return self._test_all_items(item_test)
 
-    @InvarianceItem.test_util_behaviour.used_events
+    def _test_all_items(self, item_test):
+        """
+        Apply the `item_test` function on all instances of
+        :class:`InvarianceItemBase` and aggregate the returned
+        :class:`~lisa.tests.base.ResultBundle` into one.
+
+        :attr:`~lisa.tests.base.Result.UNDECIDED` is ignored.
+        """
+        item_res_bundles = [
+            item_test(item)
+            for item in self.invariance_items
+        ]
+        return AggregatedResultBundle(item_res_bundles, 'cpu')
+
+class TaskInvariance(InvarianceBase):
```

Comment:
nit: 2 blank lines between module-level declarations

Reply:
Will change it.